lib/ArtifactUtils.cpp

```diff
@@ -0,0 +1,223 @@
+/*
+ * CArtHandler.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#include "StdInc.h"
+#include "CArtHandler.h"
+
+#include "GameSettings.h"
+#include "ArtifactUtils.h"
```

Comment:
"Main" include should be first (or well, second, after StdInc.h)
```suggestion
#include "ArtifactUtils.h"

#include "CArtHandler.h"
#include "GameSettings.h"
```